releasenotes/notes/decompose-status-further-with-subcommands-8fd7f1282e990449.yaml

```diff
@@ -7,4 +7,6 @@
 # Each section note must be formatted as reStructuredText.
 ---
 enhancements:
-  - Add `agent status component [name]` subcommand to fetch only given section status (for example, `datadog-agent status component forwarder` to get forwarder status).
+  - Add `agent status [name]` subcommand to fetch only given section status (for example, `datadog-agent status forwarder` to get forwarder status).
+deprecations:
+  - The `datadog-agent status component [name]` syntax will be replace by `datadog-agent status [name]` in Datadog Agent 7.55
```

Comment:
```suggestion
  - The `datadog-agent status component [name]` syntax will be replaced by `datadog-agent status [name]` in Datadog Agent 7.55
```

Reply:
Done in 1a5f8a7122b0199f4439066005241b293e29f8b4.